client/devdocs/design/component-playground.jsx

```diff
@@ -8,14 +8,15 @@ import classNames from 'classnames';
 import PropTypes from 'prop-types';
 import { LiveProvider, LiveEditor, LiveError, LivePreview } from 'react-live';
 import Gridicon from 'gridicons';
+import * as playgroundScopeForGutenbergComponents from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
 import Button from 'components/button';
 import ClipboardButton from 'components/forms/clipboard-button';
 import DocsExampleWrapper from 'devdocs/docs-example/wrapper';
-import * as playgroundScope from 'devdocs/design/playground-scope';
+import * as playgroundScopeForComponents from 'devdocs/design/playground-scope';
```

Comment:
🤔 not sure we can get around not importing both sets.

Reply:
What about this?
```javascript
const scope = 'gutenberg-components' === section ? require('@wordpress/components') : require('devdocs/design/playground-scope');
```

Reply:
It depends on where we put the requires. For example think of visiting http://calypso.localhost:3000/devdocs/design, then clicking on Gutenberg Components without a browser refresh.

 Maybe we could separate the two and move the playground component into an async load?

Reply:
`require` loads modules on-demand, so it shouldn't cause any issues when switching from one page to another